config/src/kura.rs

```diff
@@ -17,11 +15,11 @@ pub struct Configuration {
     /// Initialization mode: `strict` or `fast`.
     pub init_mode: Mode,
     /// Path to the existing block store folder or path to create new folder.
-    pub block_store_path: String,
-    /// Maximum number of blocks to write into a single storage file.
-    pub blocks_per_storage_file: NonZeroU64,
-    /// Default buffer capacity of actor's MPSC channel.
-    pub actor_channel_capacity: u32,
+    pub block_store_path: PathBuf,
+    // /// Maximum number of blocks to write into a single storage file.
```

Comment:
Commented code

as an aside note: curious how we ended up with these unused configuration options

Reply:
Don't know the history of `blocks_per_storage_file`. For `actor_channel_capacity` you can look here: https://github.com/hyperledger/iroha-2-docs/issues/392#issuecomment-1722872161

Reply:
Removed dead code